### PR TITLE
Allow helpers to logblock lookup

### PR DESCRIPTION
**Pull request by Legoman99573**
This makes it easier for grief reports.

### CODE REVIEW

File: plugins/PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -113,6 +113,7 @@ groups:
     - permissions.user.promote.default
     - permissions.user.rank.demote
     - permissions.user.rank.promote
+    - logblock.lookup
```
Comment (sweepies) [inline plugins/PermissionsEx/permissions.yml:116]: Add `logblock.tools.tool` and `logblock.tools.toolblock`

From the wiki:
- logblock.lookup
> Grants user access to /lb lookup (Users can't use their tools without this node!)

Comment (Legoman99573) [reply plugins/PermissionsEx/permissions.yml:116]: sorry
will resubmit it

Comment (sweepies) [reply plugins/PermissionsEx/permissions.yml:116]: You can push more commits to the branch and they'll automatically be reflected here.

Comment (Legoman99573) [reply plugins/PermissionsEx/permissions.yml:116]: pushed it ;)

**Comment by ScruffyRules** [comment]
Did this server side, please close.

**Comment by Legoman99573** [comment]
Closed upon request <3

**Pull request closed without merging**